### PR TITLE
Update judgment upload page to match new content design

### PR DESCRIPTION
**Pull request by techncl**

### CODE REVIEW

File: test/controllers/UploadControllerSpec.scala [hunk 1]
```diff
@@ -139,7 +139,7 @@ class UploadControllerSpec extends FrontEndTestHelper {
       status(uploadPage) mustBe OK
       headers(uploadPage) mustBe TreeMap("Cache-Control" -> "no-store, must-revalidate")
       contentAsString(uploadPage) must include("Upload a court judgment")
-      contentAsString(uploadPage) must include("You can upload your judgment by dragging and dropping it in the area below or by clicking 'Choose file'")
+      contentAsString(uploadPage) must include("You may now upload the court judgment you wish to transfer. You can only upload one file.")
```
Comment (TomJKing) [inline test/controllers/UploadControllerSpec.scala:142]: Is it worth also checking that the: `We only accept Microsoft Word files (.docx).` wording is also present?

**Review by TomJKing — COMMENTED** [review]
Looks good. One very minor suggestion, feel free to ignore. Thanks

File: test/controllers/UploadControllerSpec.scala (hunk 1 above)
Comment (techncl) [reply test/controllers/UploadControllerSpec.scala:142]: Hmm, we could, but we already have two lines from that page already being checked for content; should we add another?

Comment (TomJKing) [reply test/controllers/UploadControllerSpec.scala:142]: Fair enough.

**Pull request merged**